From: =?utf-8?q?Florian_M=C3=BCllner?= <fmuellner@gnome.org>
Date: Mon, 14 Feb 2022 23:37:03 +0100
Subject: build: Bump some dependency versions

Mutter and gnome-shell are released in lock-step, but I forgot to
bump the requirement accordingly.

Likewise the gsettings-desktop-schemas dependency is outdated, there
are a couple of keys we depend on that were added this cycle.

And while building should still be possible with the old gjs version,
we do require a newer version at runtime, so reflect that as well.

Origin: https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/2183
---
 .gitlab-ci.yml | 2 +-
 meson.build    | 6 +++---
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/.gitlab-ci.yml b/.gitlab-ci.yml
index e0b6bcd..02bc740 100644
--- a/.gitlab-ci.yml
+++ b/.gitlab-ci.yml
@@ -31,7 +31,7 @@ variables:
   LINT_LOG: "eslint-report.xml"
   LINT_MR_LOG: "eslint-mr-report.xml"
 
-image: registry.gitlab.gnome.org/gnome/mutter/fedora/35:x86_64-2022-01-25
+image: registry.gitlab.gnome.org/gnome/mutter/fedora/35:x86_64-2022-02-18.0
 
 workflow:
   rules:
diff --git a/meson.build b/meson.build
index ff41dde..28659eb 100644
--- a/meson.build
+++ b/meson.build
@@ -23,11 +23,11 @@ eds_req = '>= 3.33.1'
 gcr_req = '>= 3.7.5'
 gio_req = '>= 2.56.0'
 gi_req = '>= 1.49.1'
-gjs_req = '>= 1.69.2'
+gjs_req = '>= 1.71.1'
 gtk_req = '>= 3.15.0'
-mutter_req = '>= 41.0'
+mutter_req = '>= 42.beta'
 polkit_req = '>= 0.100'
-schemas_req = '>= 41.alpha'
+schemas_req = '>= 42.beta'
 startup_req = '>= 0.11'
 ibus_req = '>= 1.5.19'
 gnome_desktop_req = '>= 3.35.90'
